jwst/photom/photom.py

```diff
@@ -1032,7 +1032,8 @@ def save_area_info(self, ftab, area_fname):
                     # Note that this only copied to the first slit.
                     self.input.slits[0].area = pix_area.data
                 else:
-                    self.input.area = pix_area.data
+                    self.input.area = pix_area.data[self.input.meta.subarray.ystart: self.input.meta.subarray.ysize+1,
```

Comment:
The xstart/ystart values are 1-indexed, so you'll need to convert them to 0-indexed before using them as slices. So it should be `[xy]start - 1`. And it needs to end at `ystart+ysize-1`.